apps/mobile/src/stacks/index.tsx

```diff
@@ -38,7 +38,7 @@ export const AppStack: FC<StackScreenProps<RootParamList>> = () => {
 					/>
 					<Stack.Screen
 						name="Dashboard"
-						component={DashboardStack}
+						component={BottomTabNavigation}
```

Comment:
in this place, `DashboardStack` would be much nicer name versus `BottomTabNavigation` as we're mentioning an entire Stack/FeatureBundle instead of Navigation.